virtool/tasks/task.py

```diff
@@ -29,12 +29,12 @@ def __init__(self, app, task_id):
         self.errored = False
         self.temp_dir = virtool.utils.get_temp_dir()
 
-    async def init_db(self):
+    async def startup_db(self):
```

Comment:
Not an `aiohttp` startup function. Revert change, please.

Reply:
Done!